### ISSUE TITLE
Add documentation for source based distributions like Gentoo

### ISSUE THREAD

**Issue by Eiji7** [boydm/scenic_driver_glfw#5]
I have found solution for my second problem and described how to solve it in [forum post](https://elixirforum.com/t/problem-with-dirty-close-in-scenic/16393/7?u=eiji) for `Gentoo` and `Gentoo`-based `Linux` distributions. It would be nice if someone using beautiful sentences could describe it to not cause such troubles for others.

**Comment by boydm** [comment]
I've been following the thread on the forums. Lets confirm here what the steps are. Can you summarize them?

**Comment by Eiji7** [comment]
@boydm Sure, no problem

In `Gentoo` we do not have `*-dev` packages like in `Ubuntu`. Of course we have a bit different package names, so we have such requiements:

```
media-libs/glew
media-libs/glfw
virtual/pkgconfig
```

Just have read `*.ebuild` file. If I understand it correctly we need to disable `wayland` support for this package in order to run it under `X` window system (otherwise it does not require `X11` libraries), so for `Gentoo` and `Gentoo`-based distributions we need to add entry to `/etc/portage/package.use`:
```
media-libs/glfw -wayland
```

Also I have found this:

> GLFW natively supports Windows, macOS and Linux and other Unix-like systems. Experimental implementations for the Wayland protocol and the Mir display server are available but not yet officially supported.
> Source: https://github.com/glfw/glfw/blob/master/README.md

In short for now we should not try `scenic` on `Wayland` at least until it would have official and stable support in `media-libs/glfw`.
